service/src/test/java/org/sunbird/learner/actors/UserManagementActorTest.java

```diff
@@ -1,1770 +1,335 @@
-            (((List<Map<String, Object>>) response.get(JsonKey.ADDRESS)).get(0)).get(JsonKey.ID);
-    jobId =
-        (String)
-            (((List<Map<String, Object>>) response.get(JsonKey.JOB_PROFILE)).get(0))
-                .get(JsonKey.ID);
-    eduId =
-        (String)
-            (((List<Map<String, Object>>) response.get(JsonKey.EDUCATION)).get(0)).get(JsonKey.ID);
+    HashMap<String, Object> reqMap = getMapObject();
+    reqMap.put(JsonKey.ORGANISATION_ID, "anyOrgId");
+    reqMap.put(JsonKey.CHANNEL, "anyChannel");
+    subject.tell(
+        getRequestedObj(false, true, true, reqMap, ActorOperations.CREATE_USER), probe.getRef());
+    Response res = probe.expectMsgClass(duration("10 second"), Response.class);
+    Assert.assertTrue(null != res && res.getResponseCode() == ResponseCode.OK);
   }
 
   @Test
-  public void testGGetUserInfoWithInvalidId() {
-    TestKit probe = new TestKit(system);
-    ActorRef subject = system.actorOf(props);
-
-    Request reqObj = new Request();
-    reqObj.setOperation(ActorOperations.GET_USER_PROFILE.getValue());
-    Map<String, Object> innerMap = new HashMap<>();
-    innerMap.put(JsonKey.USER_ID, (userId + "12345"));
-    Map<String, Object> request = new HashMap<String, Object>();
-    request.put(JsonKey.USER, innerMap);
-    reqObj.setRequest(request);
+  public void testCreateuserSuccessWithoutCallerIdAndChannelANdRootOrgId() {
 
-    subject.tell(reqObj, probe.getRef());
-    ProjectCommonException exc =
-        probe.expectMsgClass(duration("200 second"), ProjectCommonException.class);
-    assertTrue(null != exc);
-  }
-
-  @Test
-  public void testGUpdateUserAddressInfo() {
     TestKit probe = new TestKit(system);
     ActorRef subject = system.actorOf(props);
-
-    Request reqObj = new Request();
-    reqObj.setOperation(ActorOperations.UPDATE_USER.getValue());
-    Map<String, Object> innerMap = new HashMap<>();
-    innerMap.put(JsonKey.LAST_NAME, "user_last_name_twice");
-    List<Map<String, Object>> list = new ArrayList<Map<String, Object>>();
-    Map<String, Object> address = new HashMap<String, Object>();
-    address.put(JsonKey.ADDRESS_LINE1, "addr line1");
-    innerMap.put(JsonKey.USER_ID, userId);
-    address.put(JsonKey.CITY, "new city");
-    address.put(JsonKey.ID, addressId);
-    list.add(address);
-    innerMap.put(JsonKey.ADDRESS, list);
-    Map<String, Object> request = new HashMap<String, Object>();
-    request.put(JsonKey.USER, innerMap);
-    reqObj.setRequest(request);
-
-    subject.tell(reqObj, probe.getRef());
-    Response res = probe.expectMsgClass(duration("200 second"), Response.class);
-    assertTrue(null != res.get(JsonKey.RESPONSE));
-    try {
-      Thread.sleep(3000);
-    } catch (InterruptedException e) {
-      ProjectLogger.log(e.getMessage(), e);
-    }
+    HashMap<String, Object> reqMap = getMapObject();
+    subject.tell(
+        getRequestedObj(false, false, true, reqMap, ActorOperations.CREATE_USER), probe.getRef());
```

Comment:
only booleans differ - rest of the code is same for above functions..please check - duplicacy